src/styles/blocks/variable.scss

```diff
@@ -0,0 +1,2 @@
+$maincolor: #{#00ACDC};
```

Comment:
name of file should be `_variables.scss`

Reply:
src/styles/index.scss
 1:1  ✖  Unexpected leading underscore in imported partial name  scss/load-no-partial-leading-underscore

1 problem (1 error, 0 warnings)

Reply:
@damian2403 you can name it `variables.scss` it doesn't really matters. the point is that you'll have a separate file for variables